instrumentation/elasticsearch/elasticsearch-api-client-8.0/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("co.elastic.clients")
+    module.set("elasticsearch-java")
+    versions.set("[7.16,)")
+    assertInverse.set(true)
+  }
+}
+
+dependencies {
+  library("co.elastic.clients:elasticsearch-java:8.0.0")
+
+  implementation(project(":instrumentation:elasticsearch:elasticsearch-rest-common:javaagent"))
+
+  testInstrumentation(project(":instrumentation:elasticsearch:elasticsearch-rest-7.0:javaagent"))
+  testInstrumentation(project(":instrumentation:apache-httpclient:apache-httpclient-4.0:javaagent"))
+  testInstrumentation(project(":instrumentation:apache-httpasyncclient-4.1:javaagent"))
+
+  testImplementation("org.apache.logging.log4j:log4j-core:2.11.0")
```

Comment:
would it make sense to use `org.slf4j:log4j-over-slf4j` instead?

Reply:
Actually not needed at all, removed it.